config/application.rb

```diff
@@ -126,6 +126,7 @@ class Application < Rails::Application
     config.maintenance_mode = ENV["MAINTENANCE_MODE"] == "true"
 
     config.report_issue_url = ENV["REPORT_ISSUE_URL"]
+    config.help_url = ENV["HELP_URL"]
```

Comment:
This will also hide the help link for all current installations since the `ENV["HELP_URL"]` for those users is "". You'll probably want to change this to `ENV["HELP_URL"].presence || "https://docs.bigbluebutton.org/greenlight/gl-overview.html"` so that it defaults to the URL if the ENV variable isn't set